coda/templates/mdstore/bag_files_download.html

```diff
@@ -0,0 +1,15 @@
+{% extends "mdstore/base.html" %}
+{% block content %}
+{% load humanize %}
```

Comment:
I'm not quite sure how to load content into my local instance, so I'm not sure how this page actually looks, nor do I know if there is a desired look for it, but if it's just a straight list of links, I don't think we really need humanize loaded in here.